types/int.go

```diff
@@ -386,6 +408,16 @@ func (i Uint) IsZero() bool {
 	return i.i.Sign() == 0
 }
 
+// IsNegative returns true if Uint is negative
```

Comment:
Huh when would a `Uint` be negative? Sounds like a panic should happen...

Reply:
lol - good point wow. just deleting for now, I just wanted to have completion, but obviously this should never be used to begin with